Lib/ctypes/test/test_byteswap.py

```diff
@@ -311,5 +311,68 @@ class S(Structure):
         s2 = struct.pack(fmt, 0x12, 0x1234, 0x12345678, 3.14)
         self.assertEqual(bin(s1), bin(s2))
 
+    def test_union_fields_unsupported_byte_order(self):
+        if sys.byteorder == "little":
+            base = BigEndianUnion
+        else:
+            base = LittleEndianUnion
+
+        class T(base):
```

Comment:
Consider writing these 6 lines as one:

```python
class T(BigEndianUnion if sys.byteorder == "little" else LittleEndianUnion):
```

Reply:
Agreed. 

Would you agree with applying the same approach to line 173: test_struct_fields_unsupported_byte_order for consistency?

Reply:
Sure. This is one of those "either way works" cases, not a big deal. The existing code likely predates the inline conditional syntax so it was never considered when written.